src/main/java/org/jitsi/jigasi/xmpp/CallControlMucActivator.java

```diff
@@ -428,16 +447,34 @@ private void updatePresenceStatusForXmppProvider(
     private void updatePresenceStatusForXmppProvider(
```

Comment:
Ugh. Why do we have `updatePresenceStatusForXmppProvider(pps)` and `updatePresenceStatusForXmppProviders()` that implement the same logic twice? :/

I know that's not related to the PR, so I don't expect you to fix it